### PR TITLE
Disable Contact Center Email

### PR DESCRIPTION
**Pull request by jacobkagon**
### Master Pivotal ticket
* https://www.pivotaltracker.com/n/projects/2481179/stories/179155265

### Child Redmine ticket(s)
* ()
* ()

### Local build result

```
bundle exec rake parallel:spec[4]
bundle exec cucumber
```

### Latest rebase/merge tag
*

---

### Data ticket(s) Followup
* (redmine links here - optional)

### Related Pull Requests
* (github links here - optional)

---

#### Peer Review
* (For code review)

#### Functional Testing
* (For testing locally)

#### Deployment
* (for release manager and/or build manager)

### CODE REVIEW

File: features/step_definitions/footer_steps.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  disable_feature(:contact_email_feature)
+end
+
+And(/the user visits the HBX home page?/) do
+  visit '/'
+  page.execute_script "window.scrollBy(0,10000)"
+end
+
+Then(/^they should see the contact email address$/) do
+  expect(page).to have_content("info@dchealthlink.com")
```
Comment (ZASMan) [inline features/step_definitions/footer_steps.rb:17]: Change this to
```
expect(page).to have_content(EnrollRegistry[:enroll_app].setting(:mail_address).item)
```

File: features/step_definitions/footer_steps.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+Given(/email address feature is enabled?/) do
```
Comment (ZASMan) [inline features/step_definitions/footer_steps.rb:3]: Get rid of these steps, use:
```
And EnrollRegistry contact_email_feature feature is enabled
```

See `resource_registry_world`

File: app/views/ui-components/bs4/v1/footers/_footer.slim [hunk 3]
```diff
@@ -11,12 +11,13 @@ footer.footer-wrapper class=("#{"pre_prod_nav_color" if ENV['ENROLL_REVIEW_ENVIR
...
-                  i.fas.fa-envelope.fa-lg
-                  | &nbsp;
-                  = Settings.contact_center.email_address
+            - if contact_center_email_address_is_enabled?
+              li.mr-3
+                = mail_to contact_center_email_address, target: '_blank' do
+                  span
+                    i.fas.fa-envelope.fa-lg
+                    | &nbsp;
+                    = Settings.contact_center.email_address
```
Comment (ZASMan) [inline app/views/ui-components/bs4/v1/footers/_footer.slim:20]: Change this to `EnrollRegistry[:enroll_app].setting(:mail_address).item`

File: app/views/ui-components/v1/layouts/_footer.html.slim [hunk 4]
```diff
@@ -12,12 +12,13 @@ footer.footer-wrapper#footer-uic class=("#{"pre_prod_nav_color" if ENV['ENROLL_R
...
-                i.fas.fa-envelope.fa-lg
-                | &nbsp;
-                = Settings.contact_center.email_address
+          - if contact_center_email_address_is_enabled?
+            li
+              = mail_to contact_center_email_address, target: '_blank' do
+                span
+                  i.fas.fa-envelope.fa-lg
+                  | &nbsp;
+                  = Settings.contact_center.email_address
```
Comment (ZASMan) [inline app/views/ui-components/v1/layouts/_footer.html.slim:21]: Change this to `EnrollRegistry[:enroll_app].setting(:mail_address).item`

File: config/client_config/me/system/config/templates/features/enroll_app/enroll_app.yml [hunk 5]
```diff
@@ -355,3 +355,6 @@ registry:
         settings:
           - key: :days
             item: 60
+      - key: contact_email_feature
```
Comment (ZASMan) [inline config/client_config/me/system/config/templates/features/enroll_app/enroll_app.yml:358]: Come up with a name like header_footer

**Review by ZASMan — CHANGES_REQUESTED** [review]

**Review by bill-transue — APPROVED** [review]
Good job incorporating Zane's feedback. Let me know when you get this rebased and we can merge. Thanks @jacobkagon!

**Pull request merged**